docs/en/01_installation.md

```diff
@@ -4,12 +4,6 @@ title: Installation
 
 # Installation
```

Comment:
Remove this heading, change the `Configuration` heading to a h1, and rename this whole page (including `title` in the metadata above) to "Configuration"